rt-scripts-optimizer.php

```diff
@@ -110,14 +110,21 @@ function rt_scripts_handler( $tag, $handle, $src ) {
 
 	// Get handle using the paths provided in the settings.
 	foreach ( $paths_option_array as $key => $script_path ) {
-		$script_path = trim( $script_path );
+		$script_path = trim( $script_path, '/' );
 		if ( empty( $script_path ) ) {
 			continue;
 		}
 
-		if ( strpos( $src, $script_path ) && ! in_array( $handle, $skip_js, true ) ) {
-			array_push( $skip_js, $handle );
-			break;
+		$regex_pattern = get_site_url();
```

Comment:
@Souptik2001 I think instead of this, we can just have Javascript validation in place so we will just put an alert in case all scripts or styles are going to be skipped or enabled from settings.